### PR TITLE
panel: Parse qcom,panel-cphy-mode

### PR DESCRIPTION
**Pull request by lujianhua**
A small part of devices use dsi cphy mode since sm8150 platform.

### CODE REVIEW

File: dtsi.py [hunk 1]
```diff
@@ -28,6 +28,11 @@ def generate_gpios(options: Options):
 def generate_panel_dtsi(p: Panel, options: Options) -> None:
 	name = p.short_id.replace('_', '-')
 	with open(f'{p.id}/panel-{name}.dtsi', 'w') as f:
+		if p.cphy_mode:
+			f.write('''\
+#include <dt-bindings/phy/phy.h>
+
+''')
```
Comment (lujianhua) [inline dtsi.py:35]: I'm not sure if it's necessary to add a "#include" here.

File: panel.py [hunk 2]
```diff
@@ -245,6 +245,10 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		if fdt.getprop_or_none(node, 'qcom,mdss-dsi-tx-eot-append') is None:
 			self.flags.append('MIPI_DSI_MODE_NO_EOT_PACKET')
 
+		self.cphy_mode = False
+		if fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None:
+			self.cphy_mode = True
```
Comment (stephan-gh) [inline panel.py:250]: `self.cphy_mode = fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None`

**Review by stephan-gh — CHANGES_REQUESTED** [review]

File: panel.py (hunk 2 above)
Comment (lujianhua) [reply panel.py]: ok, I have completed it.

File: panel.py [hunk 3]
```diff
@@ -245,6 +245,8 @@ def __init__(self, name: str, fdt: Fdt2, node: int) -> None:
 		if fdt.getprop_or_none(node, 'qcom,mdss-dsi-tx-eot-append') is None:
 			self.flags.append('MIPI_DSI_MODE_NO_EOT_PACKET')
 
+		self.cphy_mode = fdt.getprop_or_none(node, 'qcom,panel-cphy-mode') is not None
```
Comment (stephan-gh) [inline panel.py:248]: Can you move this line above `# Check DSI controller if LDO mode is needed`? Seems a bit weird stuffed in the middle of the flags. (Sorry for not noticing this earlier)

Comment (lujianhua) [reply panel.py:248]: Ok, I will rebase it.

**Review by stephan-gh — APPROVED** [review]

**Pull request merged**